### PR TITLE
typo fixes and constant file changes

### PR DESCRIPTION
**Pull request by sendilkumarn**

### CODE REVIEW

File: generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts [hunk 1]
```diff
@@ -1,14 +1,18 @@
 export function AlertService () {
     this.toast = false;
     /*jshint validthis: true */
-    this.$get = getService;
+    var service = {  getService: getService }
```
Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts]: @sendilkumarn why did you change this?

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts]: It was throwing an error that the function to return something.

Comment (deepu105) [inline generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts]: Its a provider and hence that line is required. we need to see how we can migrate this properly

Comment (sendilkumarn) [inline generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts]: probably 

`var service = {  getService: getService }
    this.$get = service;
    this.showAsToast = function(isToast) {
        this.toast = isToast;
    };
    getService.$inject = ['$timeout', '$sce', '$translate'];
    return service;`

we can write this way but why this.$get? is that advisable?
